### PR TITLE
Fix packaging rules documentation generation

### LINKED ISSUE #250: Fix docgen for packaging rules
Needs regeneration (dunno why it wasn't generated right the first time), and the docstrings need updating to format the args correctly. Will do imminently.

### PR DESCRIPTION
**Pull request by brandjon**
Fix formatting of docstrings and regenerate the pages. Fixes #250.

### CODE REVIEW

File: docs/whl.md [hunk 1]
```diff
@@ -8,6 +8,22 @@
 whl_library(<a href="#whl_library-name">name</a>, <a href="#whl_library-extras">extras</a>, <a href="#whl_library-requirements">requirements</a>, <a href="#whl_library-whl">whl</a>)
 </pre>
 
+A rule for importing `.whl` dependencies into Bazel.
+
+<b>This rule is currently used to implement `pip_import`. It is not intended to
```
Comment (aiuto) [inline docs/whl.md]: Is this the right visual? What happens with    `stuff` when you are already in ```?

Comment (brandjon) [reply docs/whl.md]: It's not in a markdown triple-backtick, it's in a Python triple quote string (in source code, not the md file).

Comment (brandjon) [reply docs/whl.md]: You can preview the md file in github -- top right ellipses -> View file.

**Comment by brandjon** [comment]
I originally tried replacing all the `<code>` tags with backticks, but it didn't render properly in args. That's a known issue in stardoc and I think it can already be fixed by upgrading to a newer stardoc and/or changing which template we use. Will file that detail away in the tracking bug.

**Pull request merged**